### PR TITLE
feat(ST-84): add create classwork buttons

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
## Description of Change

- Added classwork create button component
- Created styles
- Added Classwork container
- Made responsive button

## Associated Ticket Link(s)

- https://trello.com/c/15BaKQF0

## Related Pull Request(s)

- N/A

## Screenshots / Screen Recordings
Website View (recording):

https://github.com/user-attachments/assets/2121d1c1-4c0d-43c8-955d-75c0a7050dc4

Screenshot mobile view:
Screen Shot 2024-09-18 at 11 54 51 PM

Screenshot for tab view:
Screen Shot 2024-09-18 at 11 55 26 PM

### CODE REVIEW

File: modules/Classroom/components/ClassworkCreateButton/ClassworkCreateButton.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { Box, Button } from "@mantine/core";
+import { useDisclosure } from "@mantine/hooks";
+import { FaBook, FaPlus, FaTimes } from "react-icons/fa";
+import { LuClipboardList, LuFileEdit } from "react-icons/lu";
```
Comment (ragib-hossain-sazim) [inline modules/Classroom/components/ClassworkCreateButton/ClassworkCreateButton.tsx:4]: Just a question are we using this `react-icons/lu` library for icons throughout the app?

**Review by ragib-hossain-sazim — APPROVED** [review]

File: modules/Classroom/containers/ClassroomClassworkContainer/ClassroomClassworkContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
```
Comment (ishmam-iqbal-sazim) [inline modules/Classroom/containers/ClassroomClassworkContainer/ClassroomClassworkContainer.tsx:1]: [NIT] - Can remove this import

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Left a nit comment

File: modules/Classroom/components/ClassworkCreateButton/ClassworkCreateButton.tsx (hunk 1 above)
Comment (faysal-alam-sazim) [reply modules/Classroom/components/ClassworkCreateButton/ClassworkCreateButton.tsx:4]: Actually I am trying to find icons from font awesome, if not found then using lucid icons.

Comment (ragib-hossain-sazim) [reply modules/Classroom/components/ClassworkCreateButton/ClassworkCreateButton.tsx:4]: Please stick to only one library for icons.  We don't want to have unnecessary packages installed and increase the bundle size of the application.

@ishmam-iqbal-sazim  Let's bring this up during our standup in Monday.

**Review by ragib-hossain-sazim — APPROVED** [review]

**Pull request merged**